ethstorage/miner/worker.go

```diff
@@ -317,8 +363,7 @@ func (w *worker) notifyResultLoop() {
 func (w *worker) resultLoop() {
 	defer w.wg.Done()
 	var startTime = time.Now().Format("2006-01-02 15:04:05")
-	var succeeded, dropped int
-	errorCache := make([]miningError, 0)
+	var lastError *miningError
```

Comment:
One idea is to keep all errors in the slice and print them all when the worker exits